dashboard/src/pages/databases/query.tsx

```
@@ -0,0 +1,95 @@
+  const [editorCode, setEditorCode] = useState<string>(emptyMap);
+  const [editorCodeFnString, setEditorCodeFnString] = useState<string>(() => editorCode);
+  const [userCodeError, setUserCodeError] = useState<string | null>(null);
+
+  function editorChanged({ code }: { code: string }) {
+    setEditorCode(code);
+  }
+
+  async function runTempQuery() {
+    try {
+      // Try to evaluate the function to check for  errors
+      eval(`(${editorCode})`);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)